### DISCUSSION TITLE
Hook notification when you turn on/off (pause/unpause) a DAG from the UI

### DISCUSSION THREAD

**Discussion by salvob41** [apache/airflow#18511] [Ideas]
### Description

It would be useful if when someone decide to turn a DAG on/off (pause/unpause) to get notified with a hook/notification that a particular DAG has been turn on (maybe knowing who made this action)

### Use case/motivation

it's extremely easy turn a DAG on/off. This might get dangerous when different people (from different team) use airflow as scheduler. Having a hook might give some controls on what is being executed or not.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
How about using API calls to check the status of DAGs that should be enabled? This is much better than notification as it can be based on current state (and run periodically - even as a separate workflow). 

Converting that into a discussion as I think this is not ready yet to be a "feature" and needs some discussion. We can convert it back if we find it is needed.

**Reply by eugeneYWang** [reply]
Since airflow was running at the back stage most of the time, some teams might get used to hearing airflow events passively. Hook notifications have been applied to DAG notifications, maybe it could be used here as well

**Reply by potiuk** [reply]
Yeah. Agree it might be an interesting feature. Let's see if someone will implement it. Converted it back to feature.
